client/server-connection.cpp

```diff
@@ -84,6 +97,8 @@ void ServerConnection::Disconnect() {
 }
 
 void ServerConnection::Stop() {
+	io_work = asio::any_io_executor();
```

Comment:
Not sure what's going on here. Is it grabbing a handle to do work after the close that might still be in the pipie?

Reply:
I basically pulled the example from [The asio page on io_contexts](https://think-async.com/Asio/asio-1.18.1/doc/asio/reference/io_context.html)
and from what I understand, this "resets" the work tracking object, telling asio that's it's done.